tools/cldr-code/src/main/java/org/unicode/cldr/util/VettingViewer.java

```diff
@@ -996,10 +1103,19 @@ private void writeSummaryTable(Appendable output, String header, Level desiredLe
         if (USE_FORKJOIN) {
```

Comment:
here again, USE_FORKJOIN belongs in a class that just knows about threads and generic task management.

writeSummaryTable knows too much. The CDLR-specific knowledge (what kind of summary? what kind of table?) belongs in separate classes from the knowledge about threads.

Reply:
Forkjoin should probably be removed as dead code. I basically reimplemented it but using managed threads. All agreed.